tests/test_core.py

```diff
@@ -1914,13 +1851,13 @@ def test_biggerswitch(self):
 59899: 598995989959899
 Success!''')
 
-  @no_wasm_backend()
+  @no_wasm_backend("WebAssembly hasn't implemented computed gotos")
```

Comment:
Oh ok, so maybe another rephrasing is "wasm backend reports:" followed by that quote from it.